card-game/src/components/Button.jsx

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+import resetAll from '../utils/reset';
+
+
+export default function Button({difficulty, setAll, setCorrect, flippedList}) {
```

Comment:
이렇게 구조분해할당해보는 것은 어떨까요?
```suggestion
export default function Button(props) {
    const {difficulty, setAll, setCorrect, flippedList}=props;
```
넘겨받는 프롭스가 많다보니, 이렇게 써주는 것이 훨씬 간결하다는 생각이 들어서용!

Reply:
코드까지 쳐주는 지수갓